src/frontend/screens/WebView/index.tsx

```diff
@@ -151,6 +151,14 @@ export default function WebView() {
     if (webview) {
       const loadstop = async () => {
         setLoading({ ...loading, refresh: false })
+        const userAgent =
+          startUrl === epicLoginUrl
+            ? 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) EpicGamesLauncher'
+            : 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) Chrome/200.0'
+        if (webview.getUserAgent() != userAgent) {
+          webview.setUserAgent(userAgent)
+          console.log(userAgent)
```

Comment:
looks good, maybe get rid of this console.log tho

Reply:
forgot about that